ui/src/Components/Login/Login.js

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+export const Login = () => {
```

Comment:
We are not using this pattern for component creation. Please check the `button` or `header` to follow the same pattern.